src/Persistence/PaYaml/Models/SchemaV3/PaModule.cs

```diff
@@ -17,4 +17,6 @@ public record PaModule
     public NamedObjectMapping<DataSourceInstance>? DataSources { get; init; }
 
     public EditorStateInstance? EditorState { get; init; }
+
+    public NamedObjectMapping<ThemeDefinition>? Themes { get; init; }
```

Comment:
you'll also need to add examples to the test files to verify round-trip serialization.